app/javascript/hooks/mutations/recordings/useUpdateRecordingVisibility.jsx

```diff
@@ -0,0 +1,15 @@
+import { useMutation, useQueryClient } from 'react-query';
+import axios from 'axios';
```

Comment:
Using the custom configured `axiosInstance` can be required in the upcoming sprints.